build.sh

```diff
@@ -18,13 +18,13 @@ fetch_deps() {
 build_uefitools() {
 	[ -r .uefitools_done ] && return
 	echo " => Building UEFI tools"
-	make -C edk2/BaseTools && touch .uefitools_done
+	make -C edk2/BaseTools -j$(getconf _NPROCESSORS_ONLN) && touch .uefitools_done
```

Comment:
The edk2-platforms readme says not to do this: https://github.com/tianocore/edk2-platforms/blob/master/Readme.md?plain=1#L120

Reply:
It appears that's outdated information, since the dependencies were fixed in 2017 to allow it.
```
commit 9e1131b70b4b6002cd1904401946ef0b22cf7d33
Author: Liming Gao <liming.gao@intel.com>
Date:   Tue Nov 28 11:18:51 2017 +0800

    BaseTools: Update BaseTools top GNUMakefile with the clear dependency
    
    https://bugzilla.tianocore.org/show_bug.cgi?id=786
    
    After GNUmakefile dependency is fixed up, it can make with -j N to enable
    multiple thread build in base tools C source and save build time.
    
    In my linux host machine, make -j 4 to compile BaseTools and save ~60% time.
    
    Contributed-under: TianoCore Contribution Agreement 1.1
    Signed-off-by: Liming Gao <liming.gao@intel.com>
    Reviewed-by: Yonghong Zhu <yonghong.zhu@intel.com>
```

I'll submit a patch to edk2-platforms to fix that.

Reply:
Thanks!